full-service/src/service/sync.rs

```diff
@@ -63,24 +55,27 @@ use std::{
     thread,
 };
 
-///  The maximal number of blocks a worker thread would process at once.
-const MAX_BLOCKS_PROCESSING_CHUNK_SIZE: usize = 5;
+///  The number of blocks processed by a worker when incrementally updating an
+/// account.
+const BLOCKS_PER_ACCOUNT_UPDATE: u64 = 5;
 
 /// The AccountId corresponds to the Account's primary key: account_key_hex.
 pub type AccountId = String;
 
-/// Message type our crossbeam channel uses to communicate with the worker
-/// thread pull.
+/// Commands for the Sync worker.
 enum SyncMsg {
```

Comment:
I'd like to move these towards more idempotent units of work. For example "Sync account X against blocks 100..200" is much easier to reason about than "Sync account X against whatever blocks happen to be in the ledger when you get this message".